removed docs folder and updated readme (#1424)
evmos/evmos: README.md

It's built using the [Cosmos SDK](https://github.com/cosmos/cosmos-sdk/)
which runs on top of the [Tendermint Core](https://github.com/tendermint/tendermint) consensus engine.

> ## Docs Moved

Reviewer: ```suggestion
## Documentation
```

> We revamped our documentations and moved the work to a [separate repository](https://github.com/evmos/docs).

Reviewer: Would keep this in present tense
```suggestion
Our documentation is hosted in a [separate repository](https://github.com/evmos/docs).
```
